rainier-core/src/main/scala/com/stripe/rainier/compute/ToReal.scala

```diff
@@ -33,6 +33,16 @@ trait LowPriToReal {
 object ToReal extends LowPriToReal {
   def apply[A](a: A)(implicit toReal: ToReal[A]): Real = toReal(a)
 
+  implicit val fromInt: ToReal[Int] =
```

Comment:
how about `fromDouble`?

Reply:
That's covered by the `numeric` in the lowpri. I needed to add int/double here so that they create FractionDecimal instances.